tests/robottelo/test_report.py

```diff
@@ -0,0 +1,76 @@
+import datetime
+import glob
+import os
+from pathlib import Path
+from shutil import copyfile
+
+import pytest
+import xmltodict
+from fauxfactory import gen_string
+
+XUNIT_TIME_FORMAT = "%Y-%m-%dT%H:%M:%S"
+dummy_test_count = 2
+
+paths = [
+    ["['testsuites']['testsuite']['properties']['property']"],
+    [
+        f"['testsuites']['testsuite']['testcase'][{i}]['properties']['property']"
+        for i in range(dummy_test_count)
+    ],
+]
+
+
+@pytest.fixture(scope="session")
+def config():
+    path = Path(__file__).parent
+    conf_path = path / "../foreman"
+    copyfile(str(conf_path / '../foreman/conftest.py'), f'{str(path)}/conftest.py')
+    yield
+    os.remove(f'{str(path)}/conftest.py')
+
+
+@pytest.fixture(scope="session")
+def exec_test(config):
+    report_file = f'report_{gen_string("alphanumeric")}.xml'
+    pytest.main(
+        [
+            '-n2',
+            f'--junit-xml={report_file}',
+            f'{str(Path(__file__).parent)}/test_report.py::test_dummy',
```

Comment:
`f'{str(Path(__file__))}::test_dummy'`   <- isn't this better?

why using `.parent` when you descend then to `__file__`